docs/content/posts/2023-10-10-release-1.18.0.md

```diff
@@ -0,0 +1,542 @@
+and [source idleness timeouts](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/datastream/event-time/generating_watermarks/#dealing-with-idle-sources) 
+in pure SQL via hints. Previously, these features were only available in the DataStream API.
+
+**More Information**
+- [Documentation](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/queries/hints/)
+- [FLIP-296: Extend watermark-related features for SQL](https://cwiki.apache.org/confluence/display/FLINK/FLIP-296%3A+Extend+watermark-related+features+for+SQL)
+
+## Batch Execution Improvements
+
+### Hybrid Shuffle supports Remote Storage
+
+Hybrid Shuffle supports storing the shuffle data in remote storage. The remote storage path can be configured with the 
+option `taskmanager.network.hybrid-shuffle.remote.path`. Hybrid Shuffle uses less network memory than before by 
+decoupling the memory usage from the number of parallelisms, improving the stability and ease of use. 
+
+For more detailed 
+information, please refer to 
+
+**More Information**
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-master/docs/ops/batch/batch\_shuffle/#hybrid-shuffle)
+* [FLIP-301: Hybrid Shuffle supports Remote Storage](https://cwiki.apache.org/confluence/display/FLINK/FLIP-301%3A+Hybrid+Shuffle+supports+Remote+Storage)
+
+
+### Performance Improvements & TPC-DS Benchmark
+
+In previous releases, the community has done a lot of work to improve batch processing performance, which has led to 
+significant improvements. In this release cycle, community contributors have continued to put a lot of effort into 
+improving batch performance.
+
+#### Runtime Filter for Flink SQL
+
+Runtime filter is a common optimization to improve join performance. It is designed to dynamically generate filter 
+conditions for certain Join queries at runtime to reduce the amount of scanned or shuffled data, avoid unnecessary I/O 
+and network transmission, and speed up the query. We introduced runtime filters in Flink 1.18, and verified its 
+effectiveness through the TPC-DS benchmark, and observed up to 3x speedup for some queries by enabling this feature.
+
+#### Operator Fusion Codegen for Flink SQL
+
+Operator Fusion Codegen improves the execution performance of a query by fusing an operator DAG into a single optimized 
+operator that eliminates virtual function calls, leverages CPU registers for intermediate data and reduces the 
+instruction cache miss. As a general technical optimization, we verified its effectiveness through TPC-DS, and 
+only some batch operators completed fusion codegen support in version 1.18, getting significant performance gains on 
+some query.
+
+Note that both features are considered experimental and disabled by default in Flink 1.18. 
+They can be enabled using `table.optimizer.runtime-filter.enabled` and `able.exec.operator-fusion-codegen.enabled` 
+respectively.
```

Comment:
Can we add whether we want feedback on these features, and why they are considered experimental and some words around when these will cease to be experimental.

Reply:
Happy to add this. TBH I don't know why they are considered experimental. @lsyldliu do you know?

Reply:
@lsyldliu It will be great if you can help address the comment

Reply:
After checking the two options' definitions of source code, I think we can remove the `experimental ` description, we don't need to emphasize it. In addition, it would be better to add some content about gaining user feedback.